crates/server/openapi.yaml

```diff
@@ -700,12 +700,9 @@ components:
           enum: [sourcing]
           example: sourcing
         missingContent:
-          type: array
-          description: The array of content digests that are missing for the package record.
+          "$ref": "#/components/schemas/MissingContentMap"
+          description: The missing content for the package record.
           minItems: 1
```

Comment:
```suggestion
```

Reply:
Also, we can probably remove `missingContent` from the required properties; a `null` value would indicate no missing content.

Reply:
Is it valid to be in `state: 'sourcing'` with no `missingContent`?

Reply:
It shouldn't be; right now the server will transition directly to the `processing` state if nothing is missing.

Reply:
Oh, yeah, then I guess keeping `minItems` is fine, so disregard that comment.

Reply:
That said, I'm not entirely sure if `minItems` is valid on an object (the OpenAPI documentation mentions it only for array types).

Reply:
It's `minProperties`, apparently.